depgenerators/fileattrs/javadoc.attr

```diff
@@ -1,3 +1,3 @@
-%__javadoc_requires	%{?scl:@{javadir}-utils/scl-enable %{?scl_maven} %{scl} -- }%{_rpmconfigdir}/javadoc.req
+%__javadoc_requires	%{?scl:@{javadir}-utils/scl-enable @{scl} %{scl} -- }%{_rpmconfigdir}/javadoc@{scl_suffix}.req
```

Comment:
I would like list of SCLs to enable to be configurable, either through rpm macros or with config file. Order in which SCLs are enabled matters, and in some cases enabling `@{scl}` first may be wrong. Moreover, in current state the same SCL may be enabled more than once.